internal/bminventory/inventory.go

```diff
@@ -2987,6 +2987,31 @@ func (b *bareMetalInventory) processDiskSpeedCheckResponse(ctx context.Context,
 	return b.hostApi.SetDiskSpeed(ctx, h, diskPerfCheckResponse.Path, diskPerfCheckResponse.IoSyncDuration, exitCode, nil)
 }
 
+func (b *bareMetalInventory) updateDomainNameResolutionResponse(ctx context.Context, host *models.Host, domainResolutionResponseReporter string) error {
+	var (
+		err                      error
+		domainResolutionResponse models.DomainResolutionResponse
+	)
+	log := logutil.FromContext(ctx, b.log)
+	log.Debugf("The address for domain name resolotion on host %s are: %s", host.ID.String(), domainResolutionResponse)
```

Comment:
`domainResolutionResponse` is used before it has been assigned

Reply:
done